caps/charts.py

```diff
@@ -0,0 +1,92 @@
+from typing import TYPE_CHECKING
+
+from .models import Council, DataPoint
+
+import altair as alt
+import pandas as pd
+from charting import ChartBundle
+
+
+def multi_emissions_chart(council: "Council"):
+
+    # get just the total emissions data
+    totals = [
+        "Industry Total",
+        "Commercial Total",
+        "Public Sector Total",
+        "Transport Total",
+        "Domestic Total",
+    ]
+
+    df = (
+        DataPoint.objects.filter(council=council, data_type__name_in_source__in=totals)
+        .to_dataframe("year", "data_type__name_in_source", "value")
```

Comment:
Could add something to `to_dataframe` to use tuples to rename rather than always doing this in two steps.